assembl/static2/js/app/components/debate/thread/editPostForm.jsx

```diff
@@ -149,4 +185,8 @@ const mapStateToProps = (state) => {
   };
 };
 
-export default compose(connect(mapStateToProps), graphql(updatePostMutation))(EditPostForm);
\ No newline at end of file
+export default compose(
+  connect(mapStateToProps),
+  graphql(deletePostAttachmentMutation, { name: 'deletePostAttachment' }),
```

Comment:
Should also pass a `deleteDocumentMutation` here and use it. *MAYBE* the deleteMutation could be called on component dismount if the state has a list of deleted documents? Can make an async call without blocking the UI this way.

Reply:
@vincentfretin suggested that we don't push anything on the server until the submit button is clicked. We could use the file info that we have to create the draft-js entity. I need to do some tests to estimate if it is possible and how difficult it is